tensorflow/contrib/cmake/tf_python.cmake

```diff
@@ -224,7 +224,6 @@ add_python_module("tensorflow/python/grappler")
 add_python_module("tensorflow/python/keras")
 add_python_module("tensorflow/python/keras/activations")
 add_python_module("tensorflow/python/keras/applications")
-add_python_module("tensorflow/python/keras/applications/inception_resnet_v2")
```

Comment:
Bad merge? Deleting this line is causing the CMAKE failures.

Reply:
added this line back in - have no idea how it was ever left out